From: Benjamin Drung <benjamin.drung@ionos.com>
Date: Wed, 13 Oct 2021 16:45:54 +0200
Subject: Add missing job_name to schedule.job_status example

The module `schedule.job_status` requires a job name, but the example
does not mention that.

Forwarded: https://github.com/saltstack/salt/pull/61041
Signed-off-by: Benjamin Drung <benjamin.drung@ionos.com>
---
 salt/modules/schedule.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/salt/modules/schedule.py b/salt/modules/schedule.py
index bcd64f2..e5b3dd8 100644
--- a/salt/modules/schedule.py
+++ b/salt/modules/schedule.py
@@ -1347,7 +1347,7 @@ def job_status(name):
 
     .. code-block:: bash
 
-        salt '*' schedule.job_status
+        salt '*' schedule.job_status job_name
 
     """
 
